src/test/fuzz/tx_pool.cpp

```diff
@@ -101,9 +101,6 @@ void Finish(FuzzedDataProvider& fuzzed_data_provider, MockedTxPool& tx_pool, Cha
     if (!info_all.empty()) {
         const auto& tx_to_remove = *PickValue(fuzzed_data_provider, info_all).tx;
         WITH_LOCK(tx_pool.cs, tx_pool.removeRecursive(tx_to_remove, MemPoolRemovalReason::BLOCK /* dummy */));
-        std::vector<uint256> all_txids;
-        tx_pool.queryHashes(all_txids);
-        assert(all_txids.size() < info_all.size());
```

Comment:
Why is it ok to remove this assertion?

Reply:
maybe replace this with a call to something else and keep the test. The assertion is just checking that the size has shrunk after removing something.

Reply:
Sorry, I misunderstood the test. Replaced now with:

```cpp
        const auto new_size{WITH_LOCK(tx_pool.cs, return tx_pool.entryAll().size())};
        assert(new_size < info_all.size());
```